test/resources/invalid_spec.json

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
can you re-serialize this file to 1 line (`JSON.serialize` without args) - otherwise it skews the code size LoC stats of the repo (test/ becomes 1.4k loc more)